bokehjs/src/coffee/model.coffee

```diff
@@ -17,6 +19,39 @@ export class Model extends HasProps
       for cb in callbacks
         @listenTo(@, evt, () -> cb.execute(@))
 
+    @listenTo(@, 'change:js_event_callbacks', () -> @_update_event_callbacks)
+
+
+  _process_event : (event) ->
+    # Given an applicable event, execute the associated callback
+    if not event.constructor.event_name
+       console.warn('Event is still passed as a string: ' + event)
```

Comment:
Addressed in fdd342b. As I am working in the file, I also replaced three instances of ``console.log`` in document.coffee with ``logger.warn``.